build-logic/gradle/src/main/kotlin/convention.lifecycle.gradle.kts

```diff
@@ -0,0 +1,43 @@
+val taskGroup = "Avito android build"
+
+tasks.named<TaskReportTask>("tasks") {
+    displayGroup = taskGroup
```

Comment:
How could I see tasks from others groups?

Reply:
I don't get why it's here 😆 
Just copy pasted from an example